src/styles/_ProductDetailOrderForm.scss

```diff
@@ -28,22 +31,22 @@
     min-height: 5rem;
     margin: 10px;
     justify-content: space-evenly;
-    color: v.$primary-accent-color;
-    .details-order-form__selection-text {
-      font-size: 1.4rem;
-      margin: 5px;
-    }
+
     .details-order-form__faded {
       color: v.$secondary-accent-color;
       opacity: 0.8;
     }
     select {
       text-align: center;
-      border: 1px solid v.$primary-accent-color;
-      border-radius: 12px;
+      @include m.box-border;
       height: 3rem;
       font-size: 1.5rem;
-      color: v.$primary-accent-color;
+      @include m.body-font;
+    }
+
+    select:hover {
+      cursor: pointer;
+      border: 2px solid v.$primary-accent-color;
```

Comment:
The hover state doesn't work with phone screens. If you interact with the form on the phone screen sized window, clicks would register as a hover, resulting in persistent hover styling. I fix this by having hover styling in a `min-width: 800px)` media query (so it's activated only on wider screens).

Reply:
Per our conversation, you will fix in a separate issue. Thank you for explaining this to me :) I agree with you!